draft-ietf-httpbis-priority.md

```diff
@@ -312,6 +312,9 @@ properties of an HTTP request it receives, the server is expected to control the
 cacheability or the applicability of the cached response, by using header fields
 that control the caching behavior (e.g., Cache-Control, Vary).
 
+An endpoint that fails to parse the Priority header SHOULD use default parameter
```

Comment:
Priority _header_?  I thought "header" is now "field."

Reply:
whoops, dang